pkg/hubble/parser/threefour/parser.go

```diff
@@ -190,7 +193,7 @@ func (p *Parser) resolveNames(epID uint32, ip net.IP) (names []string) {
 	return nil
 }
 
-func filterCidrLabels(labels []string) []string {
+func filterCidrLabels(log logrus.FieldLogger, labels []string) []string {
```

Comment:
nit: We might as well make this a method like the other helpers which also access `*p Parser`

Reply:
This was my initial approach but promoting it to a method involves more changes to the tests.